day3/src/reducers/cards.js

```diff
@@ -0,0 +1,47 @@
+import {
+  CARD_FLIPPED,
+  CARDS_MATCHED,
+  GAME_RESET
+} from '../actions';
+
+const initialState = [
+  { id: 0, item: 'http://placekitten.com/200/200?image=1', matched: false, open: false },
+  { id: 1, item: 'http://placekitten.com/200/200?image=6', matched: false, open: false },
+  { id: 2, item: 'http://placekitten.com/200/200?image=4', matched: false, open: false },
+  { id: 3, item: 'http://placekitten.com/200/200?image=5', matched: false, open: false },
+  { id: 4, item: 'http://placekitten.com/200/200?image=2', matched: false, open: false },
+  { id: 5, item: 'http://placekitten.com/200/200?image=2', matched: false, open: false },
+  { id: 6, item: 'http://placekitten.com/200/200?image=1', matched: false, open: false },
+  { id: 7, item: 'http://placekitten.com/200/200?image=4', matched: false, open: false },
+  { id: 8, item: 'http://placekitten.com/200/200?image=3', matched: false, open: false },
+  { id: 9, item: 'http://placekitten.com/200/200?image=6', matched: false, open: false },
+  { id: 10, item: 'http://placekitten.com/200/200?image=3', matched: false, open: false },
+  { id: 11, item: 'http://placekitten.com/200/200?image=5', matched: false, open: false },
+];
+
+const cardReducer = (state = initialState, action) => {
+  switch (action.type) {
+  case CARD_FLIPPED:
+    return state.map(c =>
+      c.id === action.payload.id ?
+        Object.assign({}, c, { open: !c.open }) :
```

Comment:
Hm, det er det samme for meg :-) Jeg er mest vant med `Object.assign, men kanskje spread er mer intuitiv?